src/windows_uwp.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2018 Developers of the Rand project.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! Implementation for Windows UWP targets. After deprecation of Windows XP
+//! and Vista, this can superseed the `RtlGenRandom`-based implementation.
+use crate::Error;
+use core::{ffi::c_void, num::NonZeroU32, ptr, u32};
```

Comment:
Do we need `u32` here? We don't have this in `windows.rs`, which also uses `u32::max_value()`.

Reply:
Ah, I forgot to remove this leftover from the old mask code.